src/lxc/attach.c

```diff
@@ -997,6 +1016,8 @@ static int attach_child_main(void* data)
 	int fd;
 	uid_t new_uid;
 	gid_t new_gid;
+	gid_t new_groups[50];
```

Comment:
That's 65k so probably not a sane default :)

I guess you could go with an empty array and re-alloc it when adding stuff to it, that'd get you rid of the arbitrary limit and not require you to init the array at 65536.